sklearn/tests/test_pipeline.py

```diff
@@ -1983,6 +2160,8 @@ def set_request(est, method, **kwarg):
     if "fit" not in method:
         pipeline = pipeline.fit(X, y, sample_weight=sample_weight, prop=prop)
 
+    if method == "inverse_transform":
+        print("ha")
```

Comment:
```suggestion
```

Just noting while I was looking at the diff... is this left over from interactive debugging? And so should it be removed?